### ISSUE TITLE
Embedding Feature  

### ISSUE THREAD

**Issue by debasish-mohanty** [n4ze3m/page-assist#68]
Can you provide options while creating chunks based on Sentences too, which will help on better answer from the knowledge base.

**Comment by n4ze3m** [comment]
Yes, we will add that in the next release. Currently, the page is using RecursiveCharacterTextSplitter from langchain.js

**Comment by BradKML** [comment]
Seconding this but more so on dynamic context chunking, and maybe chunking with "ensemble" or overlapping chunks (since long context LLMs are here) https://docs.llamaindex.ai/en/stable/examples/retrievers/ensemble_retrieval/
